deploy/helm/kuberhealthy/templates/check-reaper.yaml

```diff
@@ -26,6 +26,7 @@ spec:
           restartPolicy: OnFailure
           serviceAccountName: check-reaper
   concurrencyPolicy: Forbid
+  startingDeadlineSeconds: {{ .Values.checkReaper.startingDeadlineSeconds }}
```

Comment:
Can we add `.Values.checkReaper.startingDeadlineSeconds` to the `values.yaml` file here?  That way we don't end up templating a blank property in the spec.

Reply:
I did add the item to `values.yaml` and left it intentionally blank since that it the default for a cronjob. There's no way to have a default value defined that would match the same behavior as a nil value. 

I was thinking of using an if-statement to only add the field when defined, but a blank spec achieves the same effect. If you think this might cause an issue rendering the template or cause undesired results, I can be a little more explicit. What do you think @integrii

Reply:
@isaaguilar super sorry for the late reply! i believe that's the way we've been enabling helm values throughout the project if there are no default values provided so it'd be great to stay consistent that way!

Reply:
I added an if-statement around `startingDeadlineSeconds` so blank properties remain out of the spec. Hope this is what we were looking for.